api/v1alpha1/clustersecurityintentbinding_types.go

```diff
@@ -1,28 +1,32 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright 2023 Authors of Nimbus
 
-package v1
+package v1alpha1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-type CwResource struct {
-	Kind        string            `json:"kind"`
-	Name        string            `json:"name"`
-	Namespace   string            `json:"namespace,omitempty"`
+type NodeSelector struct {
 	MatchLabels map[string]string `json:"matchLabels,omitempty"`
 }
 
-type CwSelector struct {
-	Resources []CwResource `json:"resources,omitempty"`
-	CEL       []string     `json:"cel,omitempty"`
+type NamespaceSelector struct {
+	MatchNames   []string `json:"matchNames,omitempty"`
+	ExcludeNames []string `json:"excludeNames,omitempty"`
+}
+
+type ClusterMatchWorkloads struct {
+	NodeSelector NodeSelector      `json:"nodeSelector,omitempty"`
+	NsSelector   NamespaceSelector `json:"nsSelector,omitempty"`
+	ObjSelector  WorkloadSelector  `json:"objSelector,omitempty"`
 }
 
 // ClusterSecurityIntentBindingSpec defines the desired state of ClusterSecurityIntentBinding
 type ClusterSecurityIntentBindingSpec struct {
-	Intents  []MatchIntent `json:"intents"`
-	Selector CwSelector    `json:"selector"`
+	Intents  []MatchIntent         `json:"intents"`
+	Selector ClusterMatchWorkloads `json:"selector,omitempty"`
```

Comment:
How can I bind/apply a ClusterSecurityIntentBinding to multiple k8s workloads/resources without creating separate ClusterSecurityIntentBindings?

Reply:
Same comment as that of [api/v1alpha1/securityintentbinding_types.go]